packages/customize-widgets/src/components/inspector/index.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import {
+	BlockInspector,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import { useInstanceId } from '@wordpress/compose';
+
+export { default as BlockInspectorButton } from './block-inspector-button';
```

Comment:
```suggestion
export BlockInspectorButton from './block-inspector-button';
```

Reply:
Didn't know we can do that! Did a quick search and found out that there're a lot of `export { default as X }` usage in the source code still though 😅

Reply:
Nope, I think it's a syntax error unfortunately 😅 .

```
> 18 | export BlockInspectorButton from './block-inspector-button';
     |        ^
  19 |
  20 | function InspectorSectionMeta( { closeInspector, inspectorTitleId } ) {
  21 | 	return (

Add @babel/plugin-proposal-export-default-from (https://git.io/vb4yH) to the 'plugins' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-export-default-from (https://git.io/vb4SO) to the 'plugins' section to enable parsing.
```

Reply:
Oh! I misread and thought this was an `import`, sorry.